deepcell/applications/multiplex_segmentation.py

```diff
@@ -40,6 +40,75 @@
 from deepcell.model_zoo import PanopticNet
 
 
+import numpy as np
+
+
+def deep_watershed_subcellular(outputs, compartment='cell', min_distance=10, maxima_threshold=0.1,
+                               cell_threshold=0.1, exclude_border=False,
+                               small_objects_threshold=0):
+    """Postprocess model output to generate predictions for distinct cellular compartments
+
+    Args:
+        outputs (list): DeepWatershed model output. A list of
+            [inner_distance, outer_distance, fgbg, pixelwise], repeated for each compartment.
+            - inner_distance: Prediction for the inner distance transform.
+            - outer_distance: Prediction for the outer distance transform.
+            - fgbg: Prediction for the foregound/background transform.
+            - pixelwise: Prediction for the interior/border/background transform.
+        compartment: which cellular compartments to generate predictions for.
+            must be one of 'cell', 'nucleus', 'both'
+        min_distance (int): Minimum allowable distance between two maxima to seed cells.
+        maxima_threshold (float): Threshold for maxima used to define watershed seeds.
+        cell_threshold (float): Threshold for mask used to define allowable area for watershed.
+        exclude_border (bool): Whether to include centroid detections at the border.
+        small_objects_threshold (int): Removes objects smaller than this size.
+
+    Returns:
+        numpy.array: Uniquely labeled mask for each compartment
```

Comment:
Can we explicitly state the order of the masks, e.g. nuclear then cell and if possible which axis is the "channel" axis?

Reply:
I think the `Application` classes automatically assume the channel axis is `-1`, unless I'm misunderstanding.